test/spec.yml

```diff
@@ -137,12 +137,12 @@
     -   in: bn deploy listtest2
     -   in: bn list
         out: |-
-             FUNCTION                  LAST DEPLOYED*
-             listtest1                 201*-*-*T*:*:*Z*
-             listtest2                 201*-*-*T*:*:*Z*
+             FUNCTION                 *LAST DEPLOYED*
+             listtest1                *201*-*-*T*:*:*Z*
```

Comment:
the asterisk is not possitioned correctly imho.
i think the last asterisk in the line should be doubled in all three lines, which will allow extra lines between.

Reply:
ok, i'm wrong it seems. we changed the asterisk to be multiline a while ago.